migration/migrating-3-4/deploying_cam.adoc

```diff
@@ -0,0 +1,27 @@
+[id='deploying-cam_{context}']
+= Deploying the Cluster Application Migration tool
```

Comment:
Consider adding initialism in parentheses on first instance of headers: "Cluster Application Migration (CAM) tool" for consistency. For example, see "OLM" under "Operators."